keylime/tenant_webapp.py

```diff
@@ -61,40 +61,40 @@ def write_error(self, status_code, **kwargs):
             lines = []
             for line in traceback.format_exception(*kwargs["exc_info"]):
                 lines.append(line)
-            common.echo_json_response(self, status_code, self._reason, lines)
+            common.echo_yaml_response(self, status_code, self._reason, lines)
         else:
-            common.echo_json_response(self, status_code, self._reason)
+            common.echo_yaml_response(self, status_code, self._reason)
 
 class MainHandler(tornado.web.RequestHandler):
     def head(self):
-        common.echo_json_response(self, 405, "Not Implemented: Use /webapp/, /agents/ or /logs/ interface instead")
+        common.echo_yaml_response(self, 405, "Not Implemented: Use /webapp/, /agents/ or /logs/ interface instead")
     def get(self):
-        common.echo_json_response(self, 405, "Not Implemented: Use /webapp/, /agents/ or /logs/  interface instead")
+        common.echo_yaml_response(self, 405, "Not Implemented: Use /webapp/, /agents/ or /logs/  interface instead")
     def put(self):
-        common.echo_json_response(self, 405, "Not Implemented: Use /webapp/, /agents/ or /logs/  interface instead")
+        common.echo_yaml_response(self, 405, "Not Implemented: Use /webapp/, /agents/ or /logs/  interface instead")
     def post(self):
-        common.echo_json_response(self, 405, "Not Implemented: Use /webapp/, /agents/ or /logs/  interface instead")
+        common.echo_yaml_response(self, 405, "Not Implemented: Use /webapp/, /agents/ or /logs/  interface instead")
     def delete(self):
-        common.echo_json_response(self, 405, "Not Implemented: Use /webapp/, /agents/ or /logs/  interface instead")
+        common.echo_yaml_response(self, 405, "Not Implemented: Use /webapp/, /agents/ or /logs/  interface instead")
 
-class WebAppHandler(BaseHandler):       
+class WebAppHandler(BaseHandler):
     def head(self):
         """HEAD not supported"""
-        common.echo_json_response(self, 405, "HEAD not supported")
-  
+        common.echo_yaml_response(self, 405, "HEAD not supported")
+
     def get(self):
         """This method handles the GET requests to retrieve status on agents for all agents in a Web-based GUI.
-        
-        Currently, only the web app is available for GETing, i.e. /webapp. All other GET uri's 
-        will return errors. 
+
+        Currently, only the web app is available for GETing, i.e. /webapp. All other GET uri's
+        will return errors.
         """
-        
-        # Get default policies for TPM/vTPM from config as suggestions to user 
+
+        # Get default policies for TPM/vTPM from config as suggestions to user
         tpm_policy = json.dumps(json.loads(config.get('tenant', 'tpm_policy')), indent=2)
         vtpm_policy = json.dumps(json.loads(config.get('tenant', 'vtpm_policy')), indent=2)
-        
+
         self.set_status(200)
-        self.set_header('Content-Type', 'text/html')
+        self.set_header('Content-Type', 'text/x-yaml; charset=utf-8')
```

Comment:
This should still be `text/html`, since we're writing out an HTML document to the browser.